### DISCUSSION TITLE
Please allow saving and copy a pinned capture

### DISCUSSION THREAD

**Discussion by elad2109** [flameshot-org/flameshot#2515] [Ideas]
After pinning a capture, please allow copying (ctrl+c) and saving (ctrl+s) the captured image.

**Comment by mmahmoudian** [comment]
This is already possible but only thorough command line:

```sh
flameshot gui --clipboard --pin
```

You can read more about other things you can do in the following link:

https://flameshot.org/docs/advanced/commandline-options/

**Reply by elad2109** [reply]
Perfect, any chance it could be configurable in the UI. 
So every capture could be with/out clipboard copy?

**Reply by mmahmoudian** [reply]
This has been discussed before, and as far as I can recall the consensus was to not clutter the UI.

Note that I'm writing this I got an idea that perhaps this can be done by holding some modifier key (e.g <kbd>Shift</kbd>) and clicking on the buttons to select multiple of them. 

Let's tag the other devs and see what they think about this: @borgmanJeremy @veracioux

**Comment by elad2109** [comment]
Or it can be an advanced custom "if this than that" in the tool's options popup.

**Reply by mmahmoudian** [reply]
> Or it can be an advanced custom "if this than that" in the tool's options popup.

I don't understand what you mean by IFTTT. The Command Line Interface (CLI) is meant to be used by users in their custom shortcuts and scripts. For instance I've wrote this short script to do some of the tasks I needed:

https://codeberg.org/mehrad/dmenu_shot

You can adopt the code and change it to your needs as it is fully written in bash with enough comments to help you understand the flow

**Reply by elad2109** [reply]
Thanks a lot,
However I want this to be my default. Meaning every pinned capture will be saved to clipboard.
I don't want to open the terminal for that.

**Reply by mmahmoudian** [reply]
Just add that the following command to get executed when a keyboard shortcut in your linux machine runs when you press PrintScreen or any other keybinding:

```sh
flameshot gui --pin --clipboard 
```

In Gnome you can follow this instructions:

https://help.gnome.org/users/gnome-help/stable/keyboard-shortcuts-set.html.en

And for KDE you can use:

https://docs.kde.org/stable5/en/khelpcenter/fundamentals/shortcuts.html
